tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -515,33 +586,36 @@ void test_sphere_factory_time_dependent() {
 
     test_sphere_construction(
         dynamic_cast<const creators::Sphere&>(*sphere), inner_radius,
-        outer_radius, sphericity, use_equiangular_map, grid_points_r_angular,
-        {7, make_array<3>(refinement_level)}, false,
+        outer_radius, sphericity, use_equiangular_map, radial_partitioning,
+        false,
         std::make_tuple(
             std::pair<std::string,
                       domain::FunctionsOfTime::PiecewisePolynomial<2>>{
                 f_of_t_name,
                 {initial_time,
                  std::array<DataVector, 3>{{{3, 0.0}, velocity, {3, 0.0}}},
                  expiration_times.at(f_of_t_name)}}),
-        std::move(grid_to_inertial_maps), std::move(time_dependencies),
-        expiration_times);
+        std::move(time_dependencies), expiration_times,
+        time_dependencies[0]->functions_of_time(), velocity);
   };
 
   std::unordered_map<std::string, double> initial_expiration_times{
-      {f_of_t_name, std::numeric_limits<double>::infinity()}};
+      {f_of_t_name, 1.0}};
+
   helper(initial_expiration_times);
   initial_expiration_times.at(f_of_t_name) = 10.0;
   helper(initial_expiration_times);
 }
 }  // namespace
 
+// [[TimeOut, 15]]
 SPECTRE_TEST_CASE("Unit.Domain.Creators.Sphere", "[Domain][Unit]") {
   domain::creators::time_dependence::register_derived_with_charm();
-  test_sphere_boundaries_equiangular();
-  test_sphere_factory_equiangular();
-  test_sphere_boundaries_equidistant();
-  test_sphere_factory_equidistant();
+  // ensure CHECK_THROWS_WITH calls are properly captured
```

Comment:
you can move these comments to the start of each function, I think that's a more helpful place to have a brief description